### PR TITLE
Code linting

### PR DESCRIPTION
**Pull request by gnott**
Fairly straightforward changes in linting, variable names, whitespace, and some return values.

Refactoring of ``copy_pdf_to_output_dir()``, which invokes the magic of the PDF decapitation, results in smaller more discrete functions for each step.

### CODE REVIEW

File: packagepoa/transform.py [hunk 1]
```diff
@@ -121,34 +127,41 @@ def add_file_to_zipfile(new_zipfile, name, new_name):
...
+
+def pdf_decap_name(new_name):
+    return "decap_" + new_name
+
+
+def pdf_details(file_title_map):
+    "find the PDF file name in the zip file map"
+    pdf_name = None
+    title = None
     for name in file_title_map.keys():
```
Comment (seanwiseman) [inline packagepoa/transform.py]: Can omit `.keys()` call as it used by default for `dict` iteration.

File: tests/test_transform.py [hunk 2]
```diff
@@ -128,10 +130,9 @@ def test_copy_pdf_to_output_dir_timed_out_pdf(self, fake_decapitate):
         fake_decapitate.side_effect = FunctionTimedOut
         doi = '10.7554/eLife.12717'
         with zipfile.ZipFile(zipfile_name, 'r') as current_zipfile:
-            with self.assertRaises(Exception) as context:
-                return_value = transform.copy_pdf_to_output_dir(file_title_map, None, doi,
-                                                                current_zipfile, POA_CONFIG)
-
+            with self.assertRaises(Exception):
```
Comment (seanwiseman) [inline tests/test_transform.py:133]: Could probably be more specific about what type of exception it's expecting to see.

Comment (gnott) [reply tests/test_transform.py:133]: It's something I tried before and gave up. I don't think the timeout exception I want happens during tests, and here it only provides a little more code coverage. I think it actually is a ``ValueError`` or something in the test scenario, whereas it is a specialised type of timeout in the actual code. I'm not even sure the timeout monitor even works and I'm tempted to remove it anyway.

**Review by seanwiseman — APPROVED** [review]

File: packagepoa/transform.py (hunk 1 above)
Comment (gnott) [reply packagepoa/transform.py]: Personally I like ``.keys()`` or using ``for key, value in ...`` over a ``dict``, to indicate the type, however the linter doesn't like the latter (if you end up not using ``value`` anywhere), and if the former is the default, I will yield to popular opinion!

**Pull request merged**